### PR TITLE
Clean up machine init interface and sync CLI initial state

### PR DESCRIPTION
**Pull request by ashley-cui**
Machine init interface now has intuitive inputs for machine init
Podman-desktop will also detect if the machine has been started/stopped from not the GUI (ie, someone uses literally `podman machine start/stop`) and use that to initialize the state of the GUI

Signed-off-by: Ashley Cui <acui@redhat.com>

### CODE REVIEW

File: Podman Desktop/Models/Machine.swift [hunk 1]
```diff
@@ -62,6 +62,27 @@ class AllMachines: ObservableObject{
         let t = type(of: lst[0].name)
             print("of type '\(t)'")
     }
+    func getRunning() -> (isRunning: Bool, machineInfo: Machine?){
+        let getRunning = AllMachines()
+        print(getRunning)
```
Comment (rhatdan) [inline Podman Desktop/Models/Machine.swift]: Are these for debug?

File: Podman Desktop/Models/Machine.swift [hunk 2]
```diff
@@ -62,6 +62,27 @@ class AllMachines: ObservableObject{
...
+            if mach.running{
+                return(true, mach)
+            }
+        }
+        return (false, nil)
+    }
+        
+    func getCLIDefault() -> Machine?{
+        let getRunning = AllMachines()
+        print(getRunning)
```
Comment (rhatdan) [inline Podman Desktop/Models/Machine.swift]: Debug?

File: Podman Desktop/Models/Shell.swift [hunk 3]
```diff
@@ -41,7 +17,7 @@ func shell(launchPath: String, arguments: [String]) throws ->  (Int32, String){
     process.standardOutput = pipe
     process.standardError = pipe
     var environment =  ProcessInfo.processInfo.environment
-        environment["PATH"] = (environment["PATH"] ?? "")+":/opt/homebrew/bin/podman:/usr/local/bin/podman"
+        environment["PATH"] = (environment["PATH"] ?? "")+":/Users/acui/go/src/github.com/containers/podman/bin/darwin:/opt/homebrew/bin/:/usr/local/bin/"
```
Comment (rhatdan) [inline Podman Desktop/Models/Shell.swift]: This looks wrong.

File: Podman Desktop/Podman_DesktopApp.swift [hunk 4]
```diff
@@ -10,6 +10,17 @@ import SwiftUI
 @main
 struct Podman_DesktopApp: App {
     @NSApplicationDelegateAdaptor(AppDelegate.self) var appDelegate
+    init() {
+            if !UserDefaults.standard.bool(forKey: "didLaunchBefore") {
+                UserDefaults.standard.set(true, forKey: "didLaunchBefore")
+                print("firsttime")
+            } else {
+                print("no")
```
Comment (rhatdan) [inline Podman Desktop/Podman_DesktopApp.swift]: debug?

File: Podman Desktop/Models/Shell.swift (hunk 3 above)
Comment (ashley-cui) [reply Podman Desktop/Models/Shell.swift]: ah, this is definetely debug

**Comment by ashley-cui** [comment]
@rhatdan Yes, print statements are debug currently, I have a PR next soon that will handle the error plumbing, since right now we're basically not handling errors yet. I'll remove the print statements for now and then in the next PR handle errors

**Comment by rhatdan** [comment]
For now I am flying blind, hopefully at some point, somone who knows their stuff will chime in.
@jwhonce Any thoughts.

**Pull request merged**